### PR TITLE
Update md-app-guard-overview.md

### PR DESCRIPTION
**Pull request by sazankha**
Added reference to Application Guard for office documentation,

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview.md [hunk 1]
```diff
@@ -49,4 +49,5 @@ Application Guard has been created to target several types of systems:
 |[Configure the Group Policy settings for Microsoft Defender Application Guard](configure-md-app-guard.md) |Provides info about the available Group Policy and MDM settings.|
 |[Testing scenarios using Microsoft Defender Application Guard in your business or organization](test-scenarios-md-app-guard.md)|Provides a list of suggested testing scenarios that you can use to test Application Guard in your organization.|
 | [Microsoft Defender Application Guard Extension for web browsers](md-app-guard-browser-extension.md) | Describes the Application Guard extension for Chrome and Firefox, including known issues, and a trouble-shooting guide |
+| [Microsoft Defender Application Guard for Microsoft Office](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/install-app-guard) | Describes Application Guard for Microsoft Office, including mimimum hardware requirements, configuration and a trouble-shooting guide |
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview.md:52]: Locale detail: this page supports other languages as well and can be linked in a language agnostic form:
```suggestion
| [Microsoft Defender Application Guard for Microsoft Office](https://docs.microsoft.com/microsoft-365/security/office-365-security/install-app-guard) | Describes Application Guard for Microsoft Office, including minimum hardware requirements, configuration and a trouble-shooting guide |
```
Also: minor typo (mimimum)

**Review by illfated — APPROVED** [review]
- [X] **Rich Diff view** for easier review of the content update: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8786/files?short_path=be530cc#diff-be530cc8ef60e0d17a2d3cfae34acf83f267a66b09135dd9be783891234e7cea
- [X] **Standard source view** for adding review comments: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8786/files

Looks reasonably good to me.

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

File: windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview.md [hunk 2]
```diff
@@ -49,4 +49,5 @@ Application Guard has been created to target several types of systems:
 |[Configure the Group Policy settings for Microsoft Defender Application Guard](configure-md-app-guard.md) |Provides info about the available Group Policy and MDM settings.|
 |[Testing scenarios using Microsoft Defender Application Guard in your business or organization](test-scenarios-md-app-guard.md)|Provides a list of suggested testing scenarios that you can use to test Application Guard in your organization.|
 | [Microsoft Defender Application Guard Extension for web browsers](md-app-guard-browser-extension.md) | Describes the Application Guard extension for Chrome and Firefox, including known issues, and a trouble-shooting guide |
+| [Microsoft Defender Application Guard for Microsoft Office](https://docs.microsoft.com/microsoft-365/security/office-365-security/install-app-guard) | Describes Application Guard for Microsoft Office, including minimum hardware requirements, configuration and a trouble-shooting guide |
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview.md:52]: Slightly overzealous completion from my side, just noticed a minor detail:
```suggestion
| [Microsoft Defender Application Guard for Microsoft Office](https://docs.microsoft.com/microsoft-365/security/office-365-security/install-app-guard) | Describes Application Guard for Microsoft Office, including minimum hardware requirements, configuration and a troubleshooting guide |
```
(my browser autocorrect did not notice it either)

Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-application-guard/md-app-guard-overview.md:52]: Sorry for deleting my previous comment. I just realized that you copied some of the text from the previous entry row, so we may as well correct that one too:
```suggestion
| [Microsoft Defender Application Guard Extension for web browsers](md-app-guard-browser-extension.md) | Describes the Application Guard extension for Chrome and Firefox, including known issues, and a troubleshooting guide |
| [Microsoft Defender Application Guard for Microsoft Office](https://docs.microsoft.com/microsoft-365/security/office-365-security/install-app-guard) | Describes Application Guard for Microsoft Office, including minimum hardware requirements, configuration, and a troubleshooting guide |
```
(Yeah, I know, we always find something more when we think it is completed.)

**Comment by illfated** [comment]
Thank you for having patience with my suggestions. This time it should be conclusive.

**Review by illfated — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
Thank you for adding my suggested corrections. The current content should be good to go.

**Review by denisebmsft — APPROVED** [review]

**Comment by denisebmsft** [comment]
Thank you @sazankha and @illfated

**Pull request merged**